loops.js

```diff
@@ -6,7 +6,9 @@
 // Hoera!
 // ==========================================
 
-
+for(let i = 0; i<3; i++) {
```

Comment:
Let op dat je de spaties tussen de entries en de operatoren consequent gebruikt:
for (let i = 0; i < 3; i++) {
}